### PR TITLE
Lazy loading is added

### LINKED ISSUE #123: Enable lazy loading to improve website performance
### Is your feature request related to a problem? Please describe.

Lazy loading is a technique that can be used to improve the performance of a website by delaying the loading of images and other resources until they are needed. This can help to reduce the initial load time of a page, which can improve the user experience and make the website more responsive.

### Describe the solution you'd like

The benefits of lazy loading include:

Reduced initial load time
Improved user experience
More responsive website
Increased SEO ranking

### Describe alternatives you've considered

_No response_

### Additional context

Assign this issue to me under GSSOC'23

### PR DESCRIPTION
**Pull request by shraddha761**
Resolves #123 

## Description

Add lazy loading to overview, Transactions, PerformanceGraph , Performers 

## Checklist 

- [x] My code adheres to the established style guidelines of the project.
- [x] I have included comments in areas that may be difficult to understand.
- [x] My changes have not introduced any new warnings.
- [x] I have conducted a self-review of my code.

### CODE REVIEW

**Comment by J0SAL** [comment]
@Abhijit25Mishra can you review this

**Comment by shraddha761** [comment]
@J0SAL please review my pr

**Comment by J0SAL** [comment]
> @J0SAL please review my pr

Let's wait for @Abhijit25Mishra to review it

File: frontend/pages/_app.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default function App({ Component, pageProps }: AppProps) {
                   <meta name="description" content="Made by Joy Almeida" />
                   <link rel="icon" href="/favicon.ico" />
                 </Head>
-                <PageNavbar />
+                <PageNavbar />
```
Comment (Abhijit25Mishra) [inline frontend/pages/_app.tsx:27]: Please note that these type of changes do not get commited. For people for review and merge your code have to spend time and make sure this is correct. 

simply put, if you make our job easier the fast things get processed.

File: frontend/pages/_app.tsx [hunk 2]
```diff
@@ -34,4 +34,4 @@ export default function App({ Component, pageProps }: AppProps) {
       </MoralisProvider>
     </React.StrictMode>
   );
-}
+}
```
Comment (Abhijit25Mishra) [inline frontend/pages/_app.tsx:37]: same problem over here, how did this even get commited ?

File: frontend/pages/_app.tsx [hunk 3]
```diff
@@ -3,13 +3,13 @@ import "bootstrap/dist/css/bootstrap.css";
...
 import type { AppProps } from "next/app";
 import Head from "next/head";
-import React from "react";
 import PageNavbar from "../components/common/PageNavbar";
 import Footer from "../components/common/Footer";
 import DataState from "../context/DataContext/dataState";
 import { SpeechProvider } from "@speechly/react-client";
 import { MoralisProvider } from "react-moralis";
 import { NotificationProvider } from "web3uikit";
+import React from "react";
```
Comment (Abhijit25Mishra) [inline frontend/pages/_app.tsx:12]: details like this (implementation specification) go to the description. Why have you changed this import from line 6 to line 12 ?

**Review by Abhijit25Mishra — CHANGES_REQUESTED** [review]

**Comment by shraddha761** [comment]
@Abhijit25Mishra apologize for all this mistake I'll make fresh pr and implement lazy loading properly

**Pull request closed without merging**